lux-lib/src/build/variables.rs

```diff
@@ -130,4 +130,12 @@ mod tests {
         let result = substitute(&[&TestVariables], input).unwrap();
         assert_eq!(result, expected);
     }
+
+    #[test]
+    fn empty_string() {
+        assert_eq!(
+            substitute(&[&TestVariables], "$(EMPTY_STRING)").unwrap(),
+            "".to_string()
+        );
+    }
```

Comment:
todo: we should also create a test for a truly empty string: `substitute(&[&TestVariables], "")`, as that was also part of the previous related bug report.